modules/ROOT/pages/to-share-an-asset.adoc

```diff
@@ -0,0 +1,62 @@
+= Share an Asset
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+:page-aliases: to-share-an-asset-with-a-user.adoc
+
+Exchange administrators and asset administrators can share an asset with another user in their organization, with the whole organization, or with an external organization.
+
+Any Exchange user can click the *Shared with me* link in the left navigation menu of Exchange to show all assets shared with their user account and all assets shared with their organization.
+
+== Share With a User
+
+If the person is not in your organization, click *Access Management* > *Users* to add them.
+
+. Click an asset to show its details.
+. Click *Share*.
+. Begin typing the name of the person you want to share this asset with, and select that person in the list. You can optionally click a filter to show only users who have Viewer, Contributor, or Admin rights.
+. Click *Add*.
+. Repeat steps 3 and 4 to share with additional users.
+
+You can also click *Share* to change who you are sharing the asset with or to stop sharing with one or more users.
+
+== Share With Your Organization
+
+Exchange administrators and asset administrators can grant viewer access for an asset to everyone in their organization.
```

Comment:
Not only viewer! You can add also admin and contributor permissions to everyone in your org 🙂

BTW: Should we include another step (in sharing with a user, and sharing with your organization), suggesting the user to change the default role(viewer)?
i.e.  3 - Select the desired role on the dropdown of the new user row.

Reply:
Good points! Updated to emphasize that viewer, contributor, and admin are options. As you suggest, I think we should have a step for selecting the viewer/contributor/admin, so I added that in the sections for sharing with a user and sharing with your org.